server/src/client.ts

```diff
@@ -0,0 +1,15 @@
+import WebSocket from 'ws'
+import { v4 as uuidv4 } from 'uuid'
+
+// TODO: probably dont need a class here and can simply use an interface / object unless we might add methods ?
+export default class Client {
+  id: string
+  name: string
+  ws: WebSocket
+
+  constructor(name: string, ws: WebSocket) {
+    this.id = uuidv4()
+    this.name = name
+    this.ws = ws
+  }
+}
```

Comment:
i think we should have a Client interface and a class to implement it if we wanted it with more functionality. current makes refactoring a pain